### PR TITLE
add chubbyphp

### PR DESCRIPTION
**Pull request by dominikzogg**

### CODE REVIEW

File: php/chubbyphp/composer.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "require": {
+    "chubbyphp/chubbyphp-framework": "^2.8",
```
Comment (waghanza) [inline php/chubbyphp/composer.json]: could you please use `~2.8.0` ?

File: php/chubbyphp/composer.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+    "nikic/fast-route": "^1.3",
```
Comment (waghanza) [inline php/chubbyphp/composer.json]: could you use `*`, adding a constraint for this add more maintenance on this project :stuck_out_tongue:

File: php/chubbyphp/composer.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+    "slim/psr7": "^1.0"
```
Comment (waghanza) [inline php/chubbyphp/composer.json]: could you use `*`, adding a constraint for this add more maintenance on this project :stuck_out_tongue:

File: php/chubbyphp/config.yaml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+framework:
+  website: https://github.com/chubbyphp/chubbyphp-framework
```
Comment (waghanza) [inline php/chubbyphp/config.yaml]: could you use `github: chubbyphp/chubbyphp-framework` ?

File: php/chubbyphp/public/index.php [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    new RouterMiddleware(new FastRouteRouter([
+        Route::get('/', 'home', new CallbackRequestHandler(
+            function () use ($responseFactory) {
+                $response = $responseFactory->createResponse();
+                $response->getBody()->write('');
+
+                return $response;
+            }
+        )),
+        Route::get('/hello/{id}', 'hello', new CallbackRequestHandler(
```
Comment (waghanza) [inline php/chubbyphp/public/index.php]: this route need to be `/user/{id}` I think

**Comment by waghanza** [comment]
@dominikzogg You also need to add `chubbyphp` in :
+  `.travis.yml` for `CI`
   + You have two solution for this :
      + manual
      + automatic (require `crystal`) `bin/make ci_config`
+ `.dependabot/confif.yml` in order to track dependencies
   + You have two solution for this :
      + manual
      + automatic (require `crystal`) `bin/make deps_config`

File: php/chubbyphp/composer.json (hunk 2 above)
Comment (dominikzogg) [reply php/chubbyphp/composer.json]: done

File: php/chubbyphp/composer.json (hunk 3 above)
Comment (dominikzogg) [reply php/chubbyphp/composer.json]: done

File: php/chubbyphp/config.yaml (hunk 4 above)
Comment (dominikzogg) [reply php/chubbyphp/config.yaml]: done

File: php/chubbyphp/public/index.php (hunk 5 above)
Comment (dominikzogg) [reply php/chubbyphp/public/index.php]: done

**Comment by dominikzogg** [comment]
@waghanza thanks for adding

**Comment by waghanza** [comment]
@dominikzogg Results are updated in https://github.com/the-benchmarker/web-frameworks#results

**Comment by dominikzogg** [comment]
@waghanza thank you

**Pull request merged**